purchase_security/models/purchase_order.py

```diff
@@ -8,28 +9,36 @@
 class PurchaseOrder(models.Model):
     _inherit = "purchase.order"
 
+    @api.model
+    def _get_default_team(self):
+        """Get team from default, from user or first team."""
```

Comment:
Please don't change the computed writable. Why do you need this? You can model it inside the compute.

Reply:
Ok, compute restored (no need to add anything extra finally).